apps/trading/components/welcome-dialog/get-started.tsx

```diff
@@ -0,0 +1,83 @@
+import classNames from 'classnames';
+import { t } from '@vegaprotocol/i18n';
+import { ExternalLink, Intent, TradingButton } from '@vegaprotocol/ui-toolkit';
+import { useVegaWallet, useVegaWalletDialogStore } from '@vegaprotocol/wallet';
+import { useEnvironment } from '@vegaprotocol/environment';
+import { useLocalStorage } from '@vegaprotocol/react-helpers';
+import * as constants from '../constants';
+import { isBrowserWalletInstalled } from '@vegaprotocol/utils';
+
+interface Props {
+  lead?: string;
+}
+
+export const GetStarted = ({ lead }: Props) => {
+  const { pubKey } = useVegaWallet();
+
+  const { VEGA_ENV, VEGA_NETWORKS } = useEnvironment();
+  const CANONICAL_URL = VEGA_NETWORKS[VEGA_ENV] || 'https://console.vega.xyz';
+
+  const [, setOnboardingViewed] = useLocalStorage(
+    constants.ONBOARDING_VIEWED_KEY
+  );
+
+  const openVegaWalletDialog = useVegaWalletDialogStore(
+    (store) => store.openVegaWalletDialog
+  );
+
+  const onButtonClick = () => {
+    openVegaWalletDialog();
+    setOnboardingViewed('true');
+  };
+
+  if (!pubKey && !isBrowserWalletInstalled()) {
+    return (
+      <div
+        className={classNames(
+          'flex flex-col bg-vega-blue-300 dark:bg-vega-blue-700 border border-vega-blue-350 dark:border-vega-blue-650 px-6 gap-4',
+          { 'py-4': lead },
+          { 'mt-8 py-6': !lead }
+        )}
+        data-testid="get-started-banner"
+      >
+        {lead && <div>{lead}</div>}
```

Comment:
`lead` and the 'Get started' text should probably be a header elements, h2 and h3

Reply:
done